wordpress-mobile/WordPress-iOS
# Update internal icon set

This PR fixes some warnings that folks are seeing after I updated the app's icon set, by ensuring that all the expected icons are included in the icon set.

<img width="651" alt="Screenshot 2021-04-30 at 20 10 58" src="https://user-images.githubusercontent.com/4780/116743024-33215600-a9f0-11eb-8774-aefb6926ecb8.png">

**To test**

* Check the AppIcon-Internal iconset to ensure no warnings are shown
* Build and run and ensure no warnings due to missing icons are shown

## Regression Notes

1. Potential unintended areas of impact

None, just updating icons that will be overwritten at build time.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
